README.md

```diff
@@ -29,7 +29,36 @@ Use this template to scaffold a new website
 7. `php artisan migrate`
 8. Create a user `php artisan make:filament-user` and follow the prompts
 9. `php artisan db:seed`
-10. `npm install && npm run dev`
+10. open `docker-compose.yml` and replace the container_name with a name of this project
+11. `docker compose up -d`
+12. `npm install && npm run dev`
+
+### How to use search
+1. Add the `IsSearchable` contract to the model
+2. Implement the required methods
```

Comment:
misschien kan je even noemen of verwijzen naar wat de required methods zijn. Ik zou in de readme graag lezen wat ik moet doen. Hier staat nu eigen "doe wat je moet doen, maar je moet zelf maar even kijken wat je precies moet doen"  :D

Reply:
Als je een interface implementeerd voegt je IDE die automatisch al toe, dus dat zou niet nodig hoeven zijn